docs/definitions/label.md

```diff
@@ -0,0 +1,14 @@
+# `ml:label`
+
+In ML, a **label** refers to the value being predicted i.e. the desired
+output of a model.
+
+While often referring to discrete values (e.g.`{"cat", "dog"}`), a label can also refer to a continuous value, for example a predicted temperature or price.
+
+**Note**
+Because labels often refer to discrete values, it may be used to designate an
```

Comment:
they instead of it? Or maybe "the term can be used to" instead of using a pronoun?